### PR TITLE
Child rows, basic edit mock up

### PR DESCRIPTION
**Pull request by Cyrelc**

### CODE REVIEW

File: app/Http/Controllers/BillController.php [hunk 1]
```diff
@@ -126,7 +126,11 @@ public function edit($id) {
     }
 
     public static function getBillsInt($input) {
+<<<<<<< 08e528f656359b7339de4411fe61356623c01098
```
Comment (MrMagaw) [inline app/Http/Controllers/BillController.php]: Adding a merge conflict? Tsk tsk tsk.

File: database/migrations/2016_05_21_000003_modify_bills_add_foreigns_table.php [hunk 2]
```diff
@@ -12,11 +12,11 @@ public function up() {
 
             $table->integer('driver_pickup_id')->unsigned()->default(0);
             $table->foreign('driver_pickup_id')
-                    ->references('id')->on('driver');
+                    ->references('id')->on('drivers');
```
Comment (MrMagaw) [inline database/migrations/2016_05_21_000003_modify_bills_add_foreigns_table.php:15]: This change, although required, shouldn't be in this commit.

File: resources/views/layouts/tables.blade.php [hunk 3]
```diff
@@ -16,51 +16,71 @@
...
+			row.child(childRow(row.data()[numCols-1])).show();
 			tr.addClass('shown');
 		}
 	});
 });
 
 </script>
 
+<script type="text/javascript">
+	function edit(){
```
Comment (MrMagaw) [inline resources/views/layouts/tables.blade.php]: An empty function?

File: resources/views/layouts/tables.blade.php [hunk 4]
```diff
@@ -16,51 +16,71 @@
...
 <link rel='stylesheet' type='text/css' href='/DataTables/extensions/Buttons/css/buttons.dataTables.min.css'>
+
+<link rel='stylesheet' type='text/css' href='/css/tables.css' />
  
 @endsection
 
 @section ('content')
 
+<?php
+	$index = 0;
```
Comment (MrMagaw) [inline resources/views/layouts/tables.blade.php]: Is this ever used?

File: database/migrations/2016_05_21_000003_modify_bills_add_foreigns_table.php (hunk 2 above)
Comment (Cyrelc) [inline database/migrations/2016_05_21_000003_modify_bills_add_foreigns_table.php:15]: Change moved to it's own commit
